prog/postgres/postgres_server_nexus.rb

```diff
@@ -236,6 +236,52 @@ def before_run
     hop_wait
   end
 
+  label def configure_metrics
+    metrics_config = postgres_server.metrics_config
+    metrics_dir = metrics_config[:metrics_dir]
+    vm.sshable.cmd("mkdir -p #{metrics_dir}")
+    vm.sshable.cmd("tee #{metrics_dir}/config.json > /dev/null", stdin: metrics_config.to_json)
+
+    metrics_service = <<SERVICE
```

Comment:
Similar to above, systemd config does not seem to have instance related information. It can be moved to image generation.

Reply:
Makes sense, I wanted to keep it here for 2 main reasons:
1. In case we need to change the collection interval, it can be easily done vs changing the image and service for each instance
2. Deploying this change is easier vs the image rollout
But I don't have a strong preference overall